components/vf-u-fullbleed/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.2.0
+
+- adds a CSS custom property for when a component with `vf-u-fullbleed` needs inline margins.
```

Comment:
Not really sure what this means? Is it easy to add an example of when/where this might be used?

Reply:
I guess it's more 'inset' spacing -- so you can have a full bleed divider but not fill all the way to the edges of the viewport -- and edge case use for sure. Will rethink how this can be demonstrated and worded.

Reply:
Ah, so something like `<hr class="vf-divider vf-u-fullbleed">` but not going 100%?

Reply:
yeah - you can make the left / right margin smaller or larger -- if you ever need to.